docs/ros-bridge.md

```diff
@@ -46,6 +46,21 @@ The contents of this message fill the essential parts of the `msr::airlib::CarAp
 - `/fsds_ros_bridge/reset` [fsds_ros_bridge/Reset](../ros/src/fsds_ros_bridge/srv/Empty.html)
  Resets car to start location.
 
+## Coordinate frames and transforms
+
+The primary frame is the `fsds/FSCar` frame.
+This frame centers the core of the car.
```

Comment:
core is a bit vague, let's call it the center of gravity?

Reply:
You are right, i added some clarification.